duden/locale/es_ES.po

```diff
@@ -53,54 +53,74 @@ msgstr "No se pudo establecer la conexión. Comprueba tu conexión a Internet."
 msgid "Unexpected HTTP response status code {}"
 msgstr "Código de estado de respuesta HTTP inesperado {}"
 
-#: main.py:480
+#: main.py:484
 msgid "Display word and its article"
 msgstr "Mostrar palabra y su artículo"
 
-#: main.py:482
+#: main.py:486
 msgid "Display the word itself"
 msgstr "Muestra la propia palabra"
 
-#: main.py:484
+#: main.py:488
 msgid "Display word article"
 msgstr "Muestra el artículo de palabra"
 
-#: main.py:486
+#: main.py:490
 msgid "Display word's part of speech"
 msgstr "Muestra la parte del habla de la palabra"
 
-#: main.py:488
+#: main.py:492
 msgid "Display commonness of the word, on scale 1 to 5"
 msgstr "Muestra la frecuencia de la palabra, del 1 al 5"
 
-#: main.py:491
+#: main.py:495
 msgid "Display the context in which the word is used"
 msgstr "Muestra el contexto en que se usa la palabra"
 
-#: main.py:494
+#: main.py:498
 msgid "Display proper word separation, each part on a separate line."
-msgstr "Muestra la adecuada separación de palabra, cada parte en una línea separada."
+msgstr ""
+"Muestra la adecuada separación de palabra, cada parte en una línea separada."
 
-#: main.py:497
+#: main.py:501
 msgid "Display word meaning overview"
 msgstr "Muestra el resumen del significado de la palabra"
 
-#: main.py:499
+#: main.py:503
 msgid "List word synonyms, each on a separate line"
 msgstr "Lista sinónimos de palabra, cada uno en una línea separada"
 
-#: main.py:501
+#: main.py:505
 msgid "Display word origin"
 msgstr "Muestra origen de palabra"
 
-#: main.py:503
+#: main.py:507
 msgid "List common word compounds"
 msgstr "Lista palabras compuestas comunes"
 
-#: main.py:505
+#: main.py:509
 msgid "List grammar forms"
 msgstr "Lista formas gramaticales"
 
-#: main.py:606
+#: main.py:512
+msgid ""
+"Display n-th result in case of multple words matching the input. Starts at 1."
+msgstr ""
+
+#: main.py:515
+msgid "Enable fuzzy word matching."
+msgstr ""
+
+#: main.py:612
 msgid "Word '{}' not found"
 msgstr "Palabra «{}» no encontrada"
+
+#: main.py:617
+msgid ""
+"Found {} matching words. Use the -r/--result argument to specify which one "
+"to display."
+msgstr ""
+
+#: main.py:629
+msgid "No result with number {}."
+msgstr ""
```

Comment:
I don't know enough spanish to translate these, but feel free to fill in the translations.